.travis.yml

```diff
@@ -0,0 +1,25 @@
+language: cpp
+cache:
+    directories:
+        - /root/.cache
+sudo: required
+services:
+    - docker
+dist: trusty
+os: linux
+env:
+    - IMAG=apolloauto/apollo:dev-x86_64-20170707_1129
+before_install:
+    - if [ ! -d "/root/.cache" ]; then sudo mkdir /root/.cache; fi
+    - sudo cp scripts/AGREEMENT.txt /root/.cache/.apollo_agreement.txt
+    - docker pull ${IMAG}
+    - docker run -d -it --name apollo_dev -v ${TRAVIS_BUILD_DIR}:/apollo -v /root/.cache:/root/.cache -w /apollo ${IMAG}
+    - docker ps 
+install: skip
+script:
+    - docker exec apollo_dev  bash -c "./apollo.sh clean"
+    - docker exec apollo_dev  bash -c "./apollo.sh build"
```

Comment:
We should definitely fix this. Unit test should be included in CI. It doesn't have to be fixed in this PR, though.

Btw, next time please add a new commit instead of amending the existing one so that the reviewer would know what change you made and the comments won't be lost. GitHub will squash all the commits into one upon merge.